components/viewer/ViewProjectRow.vue

```diff
@@ -5,7 +5,7 @@
       :styles="row.styling"
       :row-id="row.id"
     />
-    <div :class="{ 'project-row': true, hidden: !isVisible }">
+    <div v-show="isVisible" :class="{ 'project-row': true }">
```

Comment:
I do not know why this is more performant since according to the docs, this should be the same thing. I suspect there's something in either the vue backend or in devtools that can only properly handle as many hidden rows as we have when using show.